core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -45,6 +49,14 @@ public UpdateTableRequest(List<UpdateRequirement> requirements, List<MetadataUpd
     this.updates = updates;
   }
 
+  public UpdateTableRequest(
```

Comment:
This doesn't need to be public right? I think it was a mistake that the other constructor was public.

Reply:
good point, yep this can be non-public